lib/standard/ropes.nit

```diff
@@ -157,6 +157,27 @@ private class Concat
 			return new Concat(left, r + s)
 		end
 	end
+
+	redef fun copy_to_native(n, len, from, to) do
+		var remlen = len
+		var subs = new RopeSubstrings.from(self, from)
```

Comment:
There's no Rope allocation here, it's just the case where a Concat node is involved in the copy_to_native operation, since the leaves are returned sequentially by the RopeSubstrings iterator, I get them one by one to copy in the provided NativeString